dss/time_series.go

```diff
@@ -91,17 +92,22 @@ func ReadTimeSeries(filePath string, record string, outputJSON string) ([]byte,
 
 		valueTime += tSeries.timeIntervalSeconds / tSeries.timeGranularitySeconds
 
+		newCDate := C.GoString(cDate)
```

Comment:
should we remove this change? It did not have any positive impact on performance.